utils/netns/netns.go

```diff
@@ -17,9 +21,17 @@ func CreateVeth(vethNameHost, vethNameNSTemp string) error {
 	if err := netlink.LinkAdd(veth); err != nil {
 		return err
 	}
-
-	err := netlink.LinkSetUp(veth)
-	return err
+	vethSetNSStateUp := &netlink.Veth{
+		LinkAttrs: netlink.LinkAttrs{
+			Name: vethNameNSTemp,
+		},
+		PeerName: vethNameHost,
+	}
+	// Set link state to up for both host and temp,
```

Comment:
Container end UP: https://semaphoreci.com/calico/libnetwork-plugin/branches/pull-request-142/builds/18
Container end not UP: https://semaphoreci.com/calico/libnetwork-plugin/branches/pull-request-142/builds/17